docs/testing.md

```diff
@@ -126,10 +126,14 @@ Cosmos DB integration tests are run by default against a locally running [Cosmos
 
 ### Running them in the CI pipeline
 
-All integration tests should go into the [integration test workflow](../.github/workflows/integrationtests.yaml), every
+All integration tests should go into the [verify workflow](../.github/workflows/verify.yaml), every
```

Comment:
Why is it called verify? Is it a standard for something specific? 
Asking to understand why we end up with all integration tests running in verify.yaml

Reply:
That file had this name before, it contains UT, checkstyle etc. We had to move integration tests into the same workflow as UT, so that we can produce a single report from all XML files